project-api/project-api-test/src/test/resources/compiler_plugin_tests/build_context_pkg_provided_compiler_plugin/main.bal

```diff
@@ -0,0 +1,8 @@
+public function main() {
```

Comment:
Copyright text is missing in the `.bal` files in the test samples

Reply:
Fixed with 3de7f8b1faa6f9a9b2d8da400979b2cdbf80bbec